### ISSUE TITLE
remote sensing resources? 

### ISSUE THREAD

**Issue by jpiaskowski** [cran-task-views/Spatial#64]
Hello,

Would it make sense to including remote sensing-related packages in spatial? These can go in the 'Agriculture' TV if need be, but it seems like 'Spatial' is a better fit. 

here are two suggestions: 

[luna](https://github.com/rspatial/luna/)
[MODISTools](https://cran.r-project.org/package=MODISTools)

Thank you,

Julia

**Comment by jpiaskowski** [comment]
Additionally:

[sen2r](https://cran.r-project.org/package=sen2r)

**Comment by Nowosad** [comment]
There are also some other mentioned in https://github.com/r-spatial/discuss/issues/56.

@jpiaskowski would you be willing to edit the https://github.com/cran-task-views/Spatial/blob/main/Spatial.md file with your suggestions?

**Comment by jpiaskowski** [comment]
Absolutely!

**Comment by Nowosad** [comment]
Hi @jpiaskowski, is there any progress on adding these packages?

**Comment by jpiaskowski** [comment]
just made a [PR](https://github.com/cran-task-views/Spatial/pull/74)

**Comment by Nowosad** [comment]
@rsbivand what do you think about adding a new section (e.g., after "Ecological analysis") called "Remote sensing"?

**Comment by rsbivand** [comment]
@Nowosad Yes, a new subsection would be good - maybe after Interfaces to Spatial Web-Services, and move/add `rgee`, `rstac`, `rsi`, `sits` to the new section, and other relevant packages. `luna` is not yet on CRAN.

**Comment by Nowosad** [comment]
@rsbivand please take a look at https://github.com/cran-task-views/Spatial/pull/75 and merge it if everything is fine (or should we remove `luna` from the list?)

**Comment by kadyb** [comment]
You can also consider:

- [openeo](https://cran.r-project.org/web/packages/openeo/index.html)
- [RStoolbox](https://cran.r-project.org/web/packages/RStoolbox/index.html)
- [CDSE](https://cran.r-project.org/web/packages/CDSE/index.html)

**Comment by Nowosad** [comment]
@kadyb PR is welcome
